src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -517,6 +567,50 @@ export function GraphWrapper({
 							showing {graphRows.length} item{graphRows.length ? 's' : ''}
 						</span>
 					)}
+					<div className="actioncombo">
+						<button
+							type="button"
+							aria-controls="repo-columnsettings-list"
+							aria-expanded={columnSettingsExpanded}
+							aria-haspopup="listbox"
+							id="repo-actioncombo-label"
+							className="actioncombo__label"
+							role="combobox"
+							aria-activedescendant={
+								columnSettingsExpanded
+									? `repo-actioncombo-item-${reposList.findIndex(
+											item => item.path === currentRepository?.path,
+									  )}`
+									: undefined
+							}
+							onClick={() => handleToggleColumnSettings()}
+						>
+							<span className="codicon codicon-settings-gear columnsettings__icon" aria-label="Column Settings"></span>
+						</button>
+						<div
+							className="actioncombo__list"
+							id="repo-actioncombo-list"
+							role="listbox"
+							tabIndex={-1}
+							aria-labelledby="repo-actioncombo-label"
+						>
+							{
+								Object.entries(graphColumns).map(([graphZoneType, column]) => column.hideable && (
+									<span
+										className="actioncombo__item"
+										role="option"
+										data-value={graphZoneType}
+										id={`repo-actioncombo-item-${graphZoneType}`}
+										key={`repo-actioncombo-item-${graphZoneType}`}
```

Comment:
same as above for the `repo-` prefix

Reply:
Done in commit: `Fixed ids/keys prefix used for the columns combo`